weedcoco/importers/mask.py

```diff
@@ -127,11 +113,64 @@ def _image_name_to_mask(name):
                     f"No mask found at {mask_path} for image named {path.name}."
                 )
             continue
+
+        yield mask_path, path
+
+    if not all_paths:
+        raise ValueError(f"No images found in {image_dir}")
+
+
+def generate_paths_from_mask_only(mask_dir: Path, image_ext: str = "jpg"):
+    all_paths = sorted(mask_dir.glob("*.*"))
+    for mask_path in all_paths:
+        if mask_path.name.startswith("."):
+            # real glob excludes these
+            continue
+        image_file_name = f"{mask_path.stem}.{image_ext}"
+        yield mask_path, image_file_name
+
+    if not all_paths:
+        raise ValueError(f"No masks found in {mask_dir}")
+
+
+def masks_to_coco(
+    path_pairs: Iterable[Tuple[str, str]],
+    color_to_category_map: Mapping[str, str],
```

Comment:
Do we take category_color mapping as an arg, or we add an image processing step to extract all the colors and generate a mapping file?

Reply:
#467 has this functionality. I'll pull that branch into here for simplicity.